pkg/cli/mirror/create.go

```diff
@@ -117,16 +125,25 @@ func (o MirrorOptions) Create(ctx context.Context, flags *pflag.FlagSet) error {
 		return err
 	}
 
-	// Run archiver
-	if err := o.prepareArchive(ctx, cfg, backend, thisRun.Sequence, manifests, blobs); err != nil {
-		return err
-	}
+	// Catch any cancel signals
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	done := make(chan bool, 1)
+	go func() {
+		<-sigs
+		if err := rollbackMeta(); err != nil {
+			logrus.Error(err)
+		}
+		done <- true
+	}()
 
-	// Handle Committer backends.
-	if committer, isCommitter := backend.(storage.Committer); isCommitter {
```

Comment:
Does this no longer need to be called?

Reply:
I think this was added when we were intending to use the Git backend, but right now we have no concrete types that implement that interface to I just remove this line.

Reply:
Might be worth putting a comment in, but not a blocker.